distribution/packages/build.gradle

```diff
@@ -330,6 +331,8 @@ Closure commonRpmConfig(boolean oss) {
     packageGroup 'Application/Internet'
     requires '/bin/bash'
 
+    obsoletes packageName, '7.0.0', Flags.LESS
```

Comment:
Could it be that we are missing or need to improve some upgrade test ? 
It seems that lack of this should have caused an upgrade to fail.
Might be because there was no conflicts so both got installed ?

Reply:
See #39414. The problem is with yum. We could and should probably also have packaging tests that use yum, but I don't think designing and building such tests should block 7.0. I created #39494 so we do not forget; thanks for making this point.